Fitness/Models/PersonalLink.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/21/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import Foundation
+
+struct PersonalLinkObject {
+    let url: URL
+    let site: PersonalLink
+    
+    init(url: String, site: PersonalLink) {
+        self.url = URL(string: url)!
```

Comment:
Is this force cast safe? Preferably we'd use the struct's premade init and just require a URL if it can't be null. If it can the struct should maintain the optional.

Reply:
The init will take in a URL object once the backend connectivity is running, but for now since all the urls were hardcoded by me the force casts are safe.